tasks/16_repeat-string/repeat-string.js

```diff
@@ -0,0 +1,9 @@
+export function repeatStringNumTimes(str, num) {
+  if (num < 1) return '';
+  const base = str;
+  while (num > 1) {
```

Comment:
There is `repeat()` method that can be used here, but I like your approach better using `while`.

Reply:
Thank you, I went with this approach because I wanted to practice while loop.